src/coreclr/jit/importer.cpp

```diff
@@ -10098,9 +10098,68 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                             return;
                         }
 
-                        op1 = gtNewOperNode(GT_LCLHEAP, TYP_I_IMPL, op2);
-                        // May throw a stack overflow exception. Obviously, we don't want locallocs to be CSE'd.
-                        op1->gtFlags |= (GTF_EXCEPT | GTF_DONT_CSE);
+                        op1 = gtNewLclHeapNode(op2, opcodeOffs);
```

Comment:
I'll move the entire CEE_LOCALLOC importation to a separate function in a separate PR in order to simplify code-review